src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -1696,8 +1689,12 @@ TerraCore:hasAnatomicalRegion
   rdfs:domain TerraCore:BioSample ;
```

Comment:
`  rdfs:domain [
    owl:unionOf (
TerraCore:BioSample
TerraCore:SingleCell
    );
  ];`

Reply:
next checkin resolves this